commands/http/handler.go

```diff
@@ -12,10 +12,12 @@ import (
 	"strings"
 	"sync"
 
-	cmds "github.com/ipfs/go-ipfs/commands"
 	"github.com/ipfs/go-ipfs/repo/config"
 
+	cmds "github.com/ipfs/go-ipfs/commands"
```

Comment:
Separation is wrong over here.

Reply:
actually that is even supposed to be a gx import. Thanks!